content/src/scripts/libs/contentDark.js

```diff
@@ -0,0 +1,90 @@
+// themeList.js
+
+const isPresentInThemeList = () => {
+  const domain = location.origin;
+  let themeList = JSON.parse(localStorage.getItem("spantree-themelist")) || {};
+
+  return domain in themeList && themeList[domain];
+};
+
```

Comment:
Can we remove the altered top nav bar height alteration. Causes the unecessary scrollbar introduction on smaller screens. (For ref I am talking when theme is switched from default to dark)

Reply:
the whole gitlab dark theme is based around the 50px topbar height and altering will require many changes to the css. Regarding the scrollbar introduction on smaller screen, I've pushed a change on scroll bars that might help.

Reply:
Works